### PR TITLE
Fixed width of pop-up on the categories bar charts

### PR DESCRIPTION
**Pull request by anjenkin**
https://federal-spending-transparency.atlassian.net/browse/DA-6250

I could only reproduce the issue when the browser is maximized or full screen.  Kevin can reproduce it more consistently.  Would be good to test across browsers at different browser widths.

### CODE REVIEW

File: src/libs/ffg/src/spending/categories/init.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ let svg,
...
+            desc: 'The federal government reports spending by category and by agency. The top five spending categories for 2019 were Socbarcial Security with $1 trillion (24% of total spending), National Defense with $688 billion (15%), Medicare with $651 billion (15%), Health with $585 billion (13%) and Income Security with $515 billion (12%). The top five agencies by federal spending for 2019 were Department of Health and Human Services with $1.2 trillion (27% of total spending), Social Security Administration with $1.1 trillion (25%), Department of the Treasury with $689 billion (16%), Department of Defense – Military Programs with $654 billion (15%) and Department of Veterans Affairs with $200 billion (4%).'
```
Comment (brett-varney) [inline src/libs/ffg/src/spending/categories/init.js:16]: oops, replace mistake?

File: src/libs/ffg/src/spending/categories/bar/chart.js [hunk 2]
```diff
@@ -144,10 +144,12 @@ export function drawChart(data, type, _config, detail, parentWidth) {
 
     if(typeof document !== 'undefined') {
         const bodyWidth = document.body.offsetWidth;
-        if (bodyWidth < 776) {
-            calculatedWidth = bodyWidth * .9;
+        if (bodyWidth > 1500) {
+            calculatedWidth = bodyWidth * .65;
+        } else if (bodyWidth < 776) {
+            calculatedWidth = bodyWidth * .85;
```
Comment (brett-varney) [inline src/libs/ffg/src/spending/categories/bar/chart.js:150]: Even better, now it only happens (for me) when IE is almost my full screen width, around 1900 px. Is this worth worrying about? Or is there a max-size that can be set so we don't need to find a way to test for even wider screens?

**Comment by anjenkin** [comment]
@brett-varney I added another condition.  Please let me know if you approve.

**Comment by brett-varney** [comment]
Still off the edge when I'm close to full-width (which is 1920):
image

**Comment by anjenkin** [comment]
@brett-varney  Updated again.  If this doesn't work we can just merge it in without the fix for larger monitors (ie. 1900 +)

**Pull request merged**